src/modules/Packages.rb

```diff
@@ -2555,12 +2562,12 @@ def log_software_selection
     #
     # @return [Array<String>] package list
     def vnc_packages
-      packages = VNC_BASE_PACKAGES.dup
+      tags = VNC_BASE_TAGS.dup
       # At least one windowmanager must be installed (#427044)
       # If none is there, use a fallback
-      packages << DEFAULT_WM unless has_window_manager?
-      packages << "yast2-x11" if Mode.autoinst
-      packages
+      tags << DEFAULT_WM unless has_window_manager?
+      tags.concat(AUTOYAST_X11_TAGS) if Mode.autoinst
+      find_providers(tags)
```

Comment:
`find_providers` returns **all** packages providing the tag. We should choose just one package.

Normally we should get just one package providing the tag, at least for the `libyui-qt` package. But in general there might be more different providers which might even conflict with each other (e.g. `smtp_daemon` is provided by `postfix`, `sendmail`, `exim`, ... and only one of them can be installed).

We should choose only one package providing the tag (and log a warning with the candidates if more packages are found, maybe the user would prefer something else...).

That's why I'd avoid using tags for the other package lists.